app/Services/Manager.php

```diff
@@ -495,4 +495,41 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     * @param  array $params
+     * @return null|void
+     */
+    public function sendEmail($user, $contest, $params)
+    {
+        if (! isset($params)) {
+            Log::error('Gladiator\Services\Manager -- No params passed');
```

Comment:
@weerd True! the logging isn't really useful since we truncate the log file on these servers, but is helpful for QA in the meantime. I would really like us to get that development hub up at some point. :)